lib/screens/authenticate/sign_in.dart

```diff
@@ -166,17 +194,22 @@ class _SignInState extends State<SignIn> {
                           color: Theme.of(context).accentColor,
                           child: Text(
                             'Forgot Password',
-                            style: TextStyle(color: Colors.white),
+                            style: TextStyle(color: getVisibleColorOnAccentColor(context)),
                           ),
                           onPressed: () {
-                            Navigator.pushNamed(context, '/accounts/forgotpass');
+                            Navigator.pushNamed(
+                                context, '/accounts/forgotpass');
                           },
                         ),
                         SizedBox(height: 20.0),
+                        !_darkTheme ?
                         Text(
                           'Tip: Enable Dark mode from settings (icon in the AppBar).',
-                          style: TextStyle(fontSize: 20.0, fontStyle: FontStyle.italic, color: Colors.green),
-                        ),
+                          style: TextStyle(
+                              fontSize: 20.0,
+                              fontStyle: FontStyle.italic,
+                              color: Colors.green),
+                        ):Text(''),
```

Comment:
are yar ye formatting hmesha aise kyu aa rhi hai :v please make the hard wrap to 900. This is to maintain the consistancy